download_ta_bdys.py

```diff
@@ -277,8 +250,123 @@ def main():
             logger.fatal('Can not create TA output table: %s' % (str(e)))
             sys.exit(1)
     
+    #
+    # Retrieve all features from the ArcGIS endpoint
+    #
+
+    layer_output_srs = osr.SpatialReference()
+    layer_output_srs.ImportFromEPSG(layer_output_srid)
+    geojson_drv = ogr.GetDriverByName('GeoJSON')
+
+    if geojson_drv is None:
+        logger.fatal('Could not load the OGR GeoJSON driver')
+        sys.exit(1)
+    
+    # ArcGIS server capabilities
+    capabilities_uri = base_uri + '/' + str(latest_layer) + '?f=json&pretty=true'
+
+    try:
+        response = urllib2.urlopen(capabilities_uri)
+    except Exception, e:
+        logger.fatal('Could open uri %s: %s' % (capabilities_uri, str(e)))
+        sys.exit(1)
+
+    try:
+        capabilities = json.load(response)
+    except Exception, e:
+        logger.fatal('Could load json from uri %s: %s' % (capabilities_uri, str(e)))
+        sys.exit(1)
+    
+    # Get some of the capabilities
+    server_version = capabilities['currentVersion']
+    server_pagination = capabilities['advancedQueryCapabilities']['supportsPagination']
```

Comment:
Think you should check if the version number is higher than 10.3 and that the python dictionary key exist (just be to robust)